solutions/web3-fetching-strategies/types/ethers-contracts/BAYCAbi.ts

```diff
@@ -0,0 +1,1124 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
If auto generated, let's make sure it's ignored by prettier (eslint by itself isn't prettier)